src/mgr/DaemonServer.cc

```diff
@@ -1067,6 +1136,8 @@ bool DaemonServer::_handle_command(
   } else {
     m = cmdctx->m_mgr;
   }
+  MessageRef mref = std::move(m.get());
```

Comment:
Not sure this line is needed. IIUC `get()` returns a raw pointer, so `std::moving` has no effect. All that happens is creation the extra `mref` handle pointing to the same object as `m`.

If so, then it's just a nit (no blocker).